src/screens/WebScreen.tsx

```diff
@@ -23,16 +24,21 @@ export const WebScreen = ({
       inModalBrowser?: boolean;
       isExternal?: boolean;
       webUrl: string;
+      isIncognito?: boolean;
     };
   };
 }) => {
   const navigation = useNavigation();
   const { isConnected } = useContext(NetworkContext);
+  const { globalSettings = {} } = useContext(SettingsContext);
+  const { settings = {} } = globalSettings;
+  const { isIncognitoWebScreens: isIncognitoByDefault = true } = settings;
```

Comment:
why renaming?

Reply:
I thought it needed to be understandable. But it doesn't. `isIncognitoWebScreens` is clear enough.

Reply:
if you want something more understandable, than name it different from the beginning, without the need of renaming?